python/examples/core/transforms.py

```diff
@@ -536,6 +536,25 @@ def __init__(self, fun_name: str, op_name: str, result_func_name: str,
     self.pipeline = (f'builtin.func({pipeline})')
 
 
+class ApplySchedule(Transform):
+
+  def __init__(self):
+    pass
+
+  def __call__(self, module: Module, **kwargs):
+    PassManager.parse('linalg-interp-transforms').run(module)
+    self.drop_schedule_from_module(module)
+    return module
+
+  # TODO: we should not need that; PDLToLLVM and LinalgTransformToLLVM should
+  # just drop automatically?
```

Comment:
There's no PDLToLLVM or LinalgTransformToLLVM :)  Only the interpreter pass. I saw value in having the interpreter pass keep the transformation sequence as a sort of documentation/debugging aid, but it can be hidden under an option.